internal/schema/0.12/module_block.go

```diff
@@ -29,6 +29,9 @@ func moduleBlockSchema() *schema.BlockSchema {
 		},
 		Description: lang.PlainText("Module block to call a locally or remotely stored module"),
 		Body: &schema.BodySchema{
+			Extensions: &schema.BodyExtensions{
+				Count: true,
+			},
```

Comment:
```suggestion
```
Count for module blocks was [introduced in 0.13](https://github.com/hashicorp/terraform/blob/v0.13/CHANGELOG.md#0130-august-10-2020). Your change in `internal/schema/0.13/module_block.go` covers this.